qemu/tests/seabios.py

```diff
@@ -96,19 +177,18 @@ def get_list():
     boot_list = utils_misc.wait_for(get_list, timeout, 1)
 
     if not boot_list:
-        raise error.TestFail("Could not get boot entries list.")
+        test.error("Could not get boot entries list.")
 
     logging.info("Got boot menu entries: '%s'", boot_list)
     for i, v in enumerate(boot_list, start=1):
         if re.search(boot_device, v, re.I):
-            error.context("Start guest from boot entry '%s'" % v,
-                          logging.info)
+            logging.info("Start guest from boot entry '%s'" % v)
             vm.send_key(str(i))
             break
     else:
-        raise error.TestFail("Could not get any boot entry match "
-                             "pattern '%s'" % boot_device)
+        test.error("Could not get any boot entry match "
+                   "pattern '%s'" % boot_device)
 
-    error.context("Log into the guest to verify it's up")
+    logging.info("Log into the guest to verify it's up")
```

Comment:
error.context is used to mention it's a step description, you can update to error_context.context

Reply:
Currently, @error.context_aware isn't recommended, But error.context depend on @error.context_aware.